Tasks/DownloadSecureFileV1/package.json

```diff
@@ -17,8 +17,12 @@
   },
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
-    "@types/node": "^6.0.0",
-    "azure-pipelines-task-lib": "^2.8.0",
-    "azure-pipelines-tasks-securefiles-common": "1.178.0"
+    "@types/node": "^10.17.0",
+    "@types/mocha": "^5.2.7",
+    "azure-pipelines-task-lib": "^3.0.6-preview.0",
+    "azure-pipelines-tasks-securefiles-common": "2.0.4-preview"
```

Comment:
Please, change the version of securefiles-common to 2.0.3-preview, that is not affected by the issue with authorization

Reply:
Thank you! Fixed